lib/Runtime/PlatformAgnostic/Platform/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(PL_SOURCE_FILES
   Linux/UnicodeText.ICU.cpp
   Linux/NumbersUtility.cpp
   Linux/Thread.cpp
+  Linux/PerfTrace.cpp
```

Comment:
If you use a tool like dtrace on OSX, is there a pre-existing way to determine that the function at `0x7deadbeefcafe` is actually the javascript function `foo`? The `perf-<pid>.map` file provides that information because I didn't see anything that exposed it previously.